src/Compilers/Core/CodeAnalysisTest/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -529,14 +520,9 @@ public sealed override void Initialize(AnalysisContext context)
                              return;
                          }
 
-                         if (argument.ArgumentKind == ArgumentKind.DefaultValue)
-                         {
-                             operationContext.ReportDiagnostic(Diagnostic.Create(UseDefaultArgumentDescriptor, invocation.Syntax.GetLocation(), argument.Parameter.Name));
-                         }
-
                          TestAscendingArgument(operationContext, argument.Value, ref priorArgumentValue);
 
-                         if (argument.ArgumentKind == ArgumentKind.ParamArray)
+                         if (argument.Parameter.IsParams)
```

Comment:
Not sure about intent here. ParameterSymbol.IsParams doesn't guarantee that the method was used in expanded form.